Failed to import the site module
Traceback (most recent call last):
  File "/usr/lib/python3.6/site.py", line 79, in <module>
    import os
  File "/usr/lib/python3.6/os.py", line 57, in <module>
    import posixpath as path
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 674, in exec_module
  File "<frozen importlib._bootstrap_external>", line 771, in get_code
  File "<frozen importlib._bootstrap_external>", line 482, in _validate_bytecode_header
KeyboardInterrupt
